wiki/People/Beatmap_Nominators/id.md

```diff
@@ -0,0 +1,172 @@
++ Time dalam Mod Post yang telah diberikan Kudosu tidak masalah. Yang sangat penting dalam Mod Post ialah postingan date dan time dari UTC+0 yang membuktikan bahwa sudah diberikan Kudosu sesuai fakta.
+
+### Bagaimana Skor Minimal di Kalkulasi
+
++ Hal ini tergantung pada `Kudosu Score` dan total `Mod Score` pendaftar yang telah didapatkan selama bulan terakhir. Rumus Matematika diperlukan untuk menentukan nilai tersebut.
++ `Kudosu Score` ditentukan sesuai rumus sebagai berikut:
+
+![Rumus Kudosu Score](/wiki/shared/KDSCORE.png "Rumus Kudosu Score")
+
++ `Mod Score` per bulan ditentukan sesuai rumus sebagai berikut:
+
+![Rumus Mod Score](/wiki/shared/MODSCORE.png "Rumus Mod Score")
+
++ [Online Tool](http://modscore.little.moe/) ini dapat digunakan untuk mengkalkulasi total Mod Score.
++ Untuk menentukan `Required Minimum Score`, pendaftar harus menjelaskan/meringkas total `Kudosu Score` dan `Mod Score` mereka dari bulan pertama, dua, dan tiga.
++ Total keseluruhan ditetapkan sebagai berikut:
+  + osu!taiko, osu!catch and osu!mania: 1050
+  + osu!: 1100
++ Ini adalah ide bagus untuk menjaga Skor Anda tetap di atas angka-angka ini, karena jumlah total nilai dari pendaftar akan sangat berbeda nantinya.
+
+### Bagaimana potensi calon Beatmap Nominator dipilih
+
++ `Mod Score` dan `Kudosu Score` dari semua pendaftar akan diperiksa untuk memverifikasi bila mereka dapat memenuhi `Required Minimum Score`.
++ Modders yang telah memenuhi syarat akan dievaluasi oleh anggota QAT dari gamemode yang mereka pilih, dengan mempertimbangkan kriteria sebagai berikut:
+  + General attitude: Perilaku dalam modding dan perilaku lainnya yang diharapkan dari [Code of Conduct](/wiki/Ranking_Criteria/Code_of_Conduct), serta perilaku umum pada Chat/Forum.
+  + Modding abilities: Pengetahuan Umum mengenai [Kriteria Ranking](/wiki/Ranking_Criteria) dan kriteria spesifik di setiap gamemode. Kemampuan tambahan seperti Metadata, Timing, dan beberapa juga dipertimbangkan.
++ Setelah mengevaluasi setiap modder, sebuah diskusi akan dibuat apakah modder tersebut layak bergabung menjadi bagian dari Beatmap Nominator atau tidak.
+
+Quality Assurance Helper
+-------------------------
+
+Quality Assurance Helper adalah sukarelawan di dalam Beatmap Nominator yang dimana tugasnya adalah membantu Quality Assurance Team memeriksa setiap beatmap yang berstatus Qualified untuk menemukan masalah (jika ada). Jika ada, mereka akan melaporkan beatmap yang dimaksud ke thread [Report a Qualified beatmap here!](https://osu.ppy.sh/forum/t/447428). 
+
+Quality Assurance Helper sangat bebas bisa memilih beatmap apa saja yang mau diperiksa setidaknya empat beatmap per bulan. Jika ditemukan seorang anggota QAH yang tidak memenuhi standar aktivitas, hak mereka akan dicabut dan hanya bisa kembali masuk setelah cooldown selama satu bulan. Beatmap Nominator yang tidak masuk dalam masa percobaan atau masa cooldown boleh bergabung dengan Quality Assurance Helper kapan saja. 
```

Comment:
Remove `sangat` at the beginning, choose either one of `bebas` or `bisa`, and then remove `apa saja` to avoid redundancy. For example, `Quality Assurance Helper bebas memilih beatmap yang ingin diperiksa, setidaknya empat beatmap per bulan.`

Change `mau` to `ingin`, as it is more formal and befitting for a wiki article.

Add `mereka` before `hanya`, and then add `melewati masa` before the first `cooldown`?